libraries/standard/mqtt/include/iot_mqtt.h

```diff
@@ -126,6 +126,25 @@ void IotMqtt_ReceiveCallback( IotNetworkConnection_t pNetworkConnection,
                               void * pReceiveContext );
 /* @[declare_mqtt_receivecallback] */
 
+/**
+ * @brief Network close callback for the MQTT library.
+ *
+ * This function should be called by the system whenever the network connection
+ * has been closed
+ *
+ * @param[in] pNetworkConnection The network connection associated with the MQTT
+ * connection, passed by the network stack.
+ * @param[in] pNetworkConnection The network connection associated with the MQTT
+ * connection, passed by the network stack.
```

Comment:
Should be documenting `reason` instead of `pNetworkConnection` here.

Reply:
Good catch, fixed.